### DISCUSSION TITLE
Deploy git changes to org

### DISCUSSION THREAD

**Discussion by MaherSaleem** [forcedotcom/salesforcedx-vscode#5803] [Ideas]
**Is your feature request related to a problem? Please describe.**
Most of the time, I forget which files I did push to my org and which I didn't, So I go back to each **changed file in git**, and push it again to the org, which takes some time to do that

**Describe the solution you'd like**
An option to push all git changed files to my org.

**Describe alternatives you've considered**
NA

**Additional context**
You can see that we have deploy button in file explorer, but we don't have it in git changes.

https://user-images.githubusercontent.com/14930756/116526044-d6d01080-a8e1-11eb-959f-7e74f4762181.mp4

**Comment by AnanyaJha** [comment]
Hi @MaherSaleem ! Checking in to see if this is still functionality y'all are looking for? Thanks so much, this'll help with prioritizing our future work!
